README.md

```diff
@@ -26,7 +26,13 @@ following example script from `metriq-client/examples/metriq_hello.py`:
 ```python
 from metriq import MetriqClient
 
-client = MetriqClient(token="[Get this token from your web app account, and replace this string with it.]")
+You will need to create an environment variable `` to store the Metriq API key. On Linux/Unix operating systems, you can run
```

Comment:
To be clear, I'm suggesting we put this comment *outside* of the code block. So for instance, you would have:

You will need to create an environment variable `METRIQ_CLIENT_API_KEY` to store the Metriq API key. On Linux/Unix operating systems, you can run

```
export METRIQ_CLIENT_API_KEY=<the key you get from the website>
```

in your terminal. On Windows, you can similarly add this variable as an environment variable. Once you have created the environment variable, you can verify that the client works as follows:

```python
from metriq import MetriqClient

client = MetriqClient(token=os.environ["METRIQ_CLIENT_API_KEY"])
print(client.hello())
>>> {'status': 'API is working', 'message': 'This is the Metriq public REST API.'}
```